federation/registry/cluster/strategy_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	genericapirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/kubernetes/federation/apis/federation"
 	"k8s.io/kubernetes/pkg/api"
+	_ "k8s.io/kubernetes/pkg/api/testapi"
```

Comment:
Along with this move: `pkg/api/testing/fuzzer.go` to `pkg/api/fuzzer/fuzzer.go`, the import is also moved to `pkg/api/fuzzer/fuzzer.go`(remember the EnvVar randomization?).

as a result, importing `pkg/api/testing` does not imply importing `pkg/api/testapi` any longer.